main/ui/animators.py

```diff
@@ -40,12 +39,20 @@ def draw(self, ctx, width, height):
             ctx.set_line_width(6)
             ctx.set_line_cap(cairo.LINE_CAP_ROUND)
             if i % 2 == 0:
-                ctx.move_to(width / 2 + i * 10, height / 2 + 3 - 8 * math.sin(self.tc + i))
-                ctx.line_to(width / 2 + i * 10, height / 2 - 3 + 8 * math.sin(self.tc + i))
+                ctx.move_to(
```

Comment:
These changes involving mathematical operations look a bit weird when moved to individual lines. Can revert them to as they were?

Reply:
I have revert them as shown in below commits, Kindly review it :)